purview/models/koji.py

```diff
@@ -34,3 +35,9 @@ class KojiTask(StructuredNode):
     priority = IntegerProperty()
     arch = StringProperty(required=True)
     method = StringProperty(required=True)
+
+
+class KojiTag(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    builds = RelationshipTo('KojiBuild', 'TAGGED_TO')
```

Comment:
Based on our discussion, this should likely be called `CONTAINED_BY` for more generic relationship names.